apps/webapp/app/v3/services/triggerScheduledTask.server.ts

```diff
@@ -7,6 +7,18 @@ import { stringifyIO } from "@trigger.dev/core/v3";
 import { nextScheduledTimestamps } from "../utils/calculateNextSchedule.server";
 import { findCurrentWorkerDeployment } from "../models/workerDeployment.server";
 import { logger } from "~/services/logger.server";
+import { env } from "~/env.server";
+import Redis from "ioredis";
+import { devPresence, DevPresence } from "~/presenters/v3/DevPresence.server";
+
+const redis = new Redis({
+  port: env.RUN_ENGINE_DEV_PRESENCE_REDIS_PORT ?? undefined,
+  host: env.RUN_ENGINE_DEV_PRESENCE_REDIS_HOST ?? undefined,
+  username: env.RUN_ENGINE_DEV_PRESENCE_REDIS_USERNAME ?? undefined,
+  password: env.RUN_ENGINE_DEV_PRESENCE_REDIS_PASSWORD ?? undefined,
+  enableAutoPipelining: true,
+  ...(env.RUN_ENGINE_DEV_PRESENCE_REDIS_TLS_DISABLED === "true" ? {} : { tls: {} }),
+});
```

Comment:
This should be removed from here now, right?

Reply:
Good spot, removed 👍